services/gallery-login.ts

```diff
@@ -17,12 +17,11 @@ export const GalleryLoginProvider = CredentialsProvider({
 
       const data = await response.json();
 
-      return {
-        id: data._id,
-        ...data,
-      };
+      return { id: data.id };
     } catch (error) {
       console.log(error);
```

Comment:
Please use the error handler function to handle errors here. Thanks

Reply:
i will throw an error because if i return the  handleErrorEdgeCases typescript will show error

Reply:
Alright, please keep track of the behavior further down the link